packages/studioCMS/src/integrations/studioCMSDashboard/utils/webVital.ts

```diff
@@ -0,0 +1,22 @@
+// @ts-expect-error - Some types can only be imported from the Astro runtime
+import { AstrojsWebVitals_Metric, db, eq } from 'astro:db';
+
+export type WebVitalsResponseItem = {
+    id: string;
+    pathname: string;
+    route: string;
+    name: string;
+    value: number;
+    rating: string;
+    timestamp: Date;
+};
+
+export async function getWebVitals(): Promise<WebVitalsResponseItem[]> {
+    const webVitals: WebVitalsResponseItem[] = await db.select()
+            .from(AstrojsWebVitals_Metric)
+            .catch(() => {
```

Comment:
Do we need to handle anything else here? Might be worth a non-verbose console log to say this call has errored

Reply:
Optional integration I just needed it not to throw an error that would crash the session if the tables where missing